docs/src/content/docs/fr/reference/plugins.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Référence des modules d'extension
+description: Une vue d'ensemble de l'API des modules d'extension Starlight.
+tableOfContents:
+  maxHeadingLevel: 4
+---
+
+Les modules d'extension Starlight peuvent personnaliser la configuration, l'interface utilisateur et le comportement de Starlight, tout en étant faciles à partager et à réutiliser.
+Cette page de référence documente l'API à laquelle ces modules d'extension ont accès.
+
+Consultez la [référence de configuration](/fr/reference/configuration/#plugins) pour en savoir plus sur l'utilisation d'un module d'extension Starlight.
+
+## Référence rapide de l'API
+
+Un module d'extension Starlight a la forme suivante.
+Voir ci-dessous pour plus de détails sur les différentes propriétés et paramètres des hooks.
```

Comment:
[Reference](https://fr.wikipedia.org/wiki/Hook_(informatique)) for keeping "hook" as other would be literal translations without a lot of meaning in this context.